http/mocks/things.go

```diff
@@ -31,13 +28,6 @@ func NewThingsClient(data map[string]string) mainflux.ThingsServiceClient {
 func (tc thingsClient) CanAccessByKey(ctx context.Context, req *mainflux.AccessByKeyReq, opts ...grpc.CallOption) (*mainflux.ThingID, error) {
 	key := req.GetToken()
 
-	// Since there is no appropriate way to simulate internal server error,
-	// we had to use this obscure approach. ErrorToken simulates gRPC
-	// call which returns internal server error.
-	if key == ServiceErrToken {
-		return nil, status.Error(codes.Internal, "internal server error")
-	}
-
```

Comment:
I'd like to keep both empty and invalid key cases.

Reply:
ok